internal/app/machined/internal/phase/services/start_services.go

```diff
@@ -44,15 +44,15 @@ func (task *StartServices) loadSystemServices(data *userdata.UserData, mode runt
 		&services.MachinedAPI{},
 		&services.Containerd{},
 		&services.Networkd{},
-		&services.Udevd{},
 		&services.OSD{},
-		&services.NTPd{},
 	)
 
 	if mode != runtime.Container {
 		// udevd-trigger is causing stalls/unresponsive stuff when running in local mode
```

Comment:
should we update comment with the reasons behind disabling these?

Reply:
+1 We should.